assets/xml/scenes/overworld/spot18.xml

```diff
@@ -1,6 +1,12 @@
 <Root>
     <File Name="spot18_scene">
         <Scene Name="spot18_scene">
+            <CutsceneHint Name="gGoronCityDaruniaCorrectCs" Offset="0x59E0"/>
+
+            <!-- For some reason, this name (`gGoronCityDarunia01Cs`) is ignored, and `spot18_sceneCutsceneData0x006930` is used instead. -->
+            <CutsceneHint Name="gGoronCityDarunia01Cs" Offset="0x6930"/>
```

Comment:
we could try to take a look at zap and see if this is an easy fix. would be good to sort this out before merging this one specifically

Reply:
Good idea. I'll take a look in ZAPD

Reply:
I think I found a fix. I'll do a PR in ZAPD.